html5/default/vm/index.js

```diff
@@ -82,4 +82,37 @@ export default function Vm (
   build(this)
 }
 
+Vm.prototype.destroy = function () {
```

Comment:
Recommended to use `$destroy()` instead to avoid common property names because the Vm properties would be defined from developers by options.